amarachukwu-agbo/hello-books
# #157879438 Document API endpoints using Swagger

#### What does this PR do?
Add swagger documentation for the API endpoints.

#### Description of Task to be completed?
- document requests and all error responses for the API endpoints
- set up swagger-ui-express to render the API documentation

#### How should this be manually tested?

Clone the repo and install application dependencies. Run command  `npm run start:server`. Navigate to the browser on ```127.0.0.1:3000/api/v1/docs```. You should see the API documentation.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

#157879438

#### Screenshots (if appropriate)

N/A

#### Questions: N/A]
